docpage/docs/Users.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 2
 
 # Users Guide
 
-Here, you will find an explanation of the main concepts of DCNiOS, such as the DCNiOS commands, how to define a workflow, involved sections, and the commun sections for all the third-party connections.
+This page explains the main concepts of DCNiOS, such as the DCNiOS commands and workflow definition.
```

Comment:
This page explains the main concepts of DCNiOS, such as the DCNiOS commands and the definition of workflows.